src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -649,7 +649,7 @@ void GameObject::Update(uint32 diff)
                     Unit* target = nullptr;
 
                     /// @todo this hack with search required until GO casting not implemented
-                    if (Unit* owner = GetOwner())
```

Comment:
Can this maybe be done without the objectaccessor lookup?

Reply:
No idea. Someone else would have to chime in. @Shauren ?

Reply:
Comment below makes it suspicious, maybe it was intended to pass `owner` as first arg to `checker` constructor?

Reply:
Ref 862097a63351e957fd4aa6cb82fe7d9edf4d3926

Reply:
objectaccessor lookup is fine